sympy/printing/pretty/pretty.py

```diff
@@ -1871,6 +1871,8 @@ def _print_Range(self, s):
         elif s.stop.is_infinite:
             it = iter(s)
             printset = next(it), next(it), dots
+        elif s.size.has(Symbol):
+            printset = s.inf, s.inf + s.step, dots, s.sup
```

Comment:
@smichr I thought of this printing symbolic `Range`. Any ideas or code suggestions?

Reply:
Should be fine for non-infinite start. If you still have an infinite start, however, you will want to switch the 2nd and 3rd (dots) arguments.

Reply:
> Should be fine for non-infinite start.

Infinite start is disallowed, so this would be the case always, isn't it?